src/deploy/functions/tasks.ts

```diff
@@ -116,6 +116,16 @@ export function createFunctionTask(
         params.errorHandler.record("warning", fnName, "make public", err.message);
       }
     }
+    if ("concurrency" in fn) {
+      if (fn.platform === "gcfv1") {
+        logger.debug("Cannot set concurrency for a v1 function");
```

Comment:
Feels like this ought to be a "this should never happen" scenario, because we should detect and error out a deployment if `concurrency` is set for a v1 function in the `prepare` stage, no?